### PR TITLE
feat: sync to libovsdb with reconnection

### PR DESCRIPTION
**Pull request by wu-cl**

### CODE REVIEW

File: pkg/agent/datapath/multiBridgeDatapath.go [hunk 1]
```diff
@@ -184,7 +184,6 @@ type DpManager struct {
 	FlowIDToRules             map[uint64]*EveroutePolicyRuleEntry
 	flowReplayChan            chan struct{}
```
Comment (zwtop) [inline pkg/agent/datapath/multiBridgeDatapath.go:185]: nit: remove unused variable `flowReplayChan`

**Comment by KeyongSun** [comment]
Please update the self-testing details.

File: pkg/agent/datapath/multiBridgeDatapath.go (hunk 1 above)
Comment (wu-cl) [reply pkg/agent/datapath/multiBridgeDatapath.go:185]: flow replay event need to refactor, del this in other patch

**Comment by wu-cl** [comment]
@KeyongSun add restarter for ovs in e2e test  
cc @zwtop

**Review by echkenluo — APPROVED** [review]

**Review by KeyongSun — APPROVED** [review]

**Pull request merged**